src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -477,83 +477,77 @@ CLASS zcl_abapgit_gui_page_repo_over IMPLEMENTATION.
+    lo_toolbar->add( iv_txt      = |Patch|
+                     iv_act      = |{ zif_abapgit_definitions=>c_action-go_patch }{ lc_dummy_key }|
+                     iv_class    = |{ lc_action_class } { lc_online_class }|
+                     iv_li_class = |{ lc_action_class }| ).
 
-    ii_html->add( action_link( lv_stage_link && lc_separator ) ).
+    lo_toolbar->add( iv_txt      = |Diff|
+                     iv_act      = |{ zif_abapgit_definitions=>c_action-go_repo_diff }{ lc_dummy_key }|
+                     iv_class    = |{ lc_action_class } { lc_online_class }|
+                     iv_li_class = |{ lc_action_class }| ).
 
-    lv_patch_link = ii_html->a(
-      iv_txt   = |Patch|
-      iv_act   = |{ zif_abapgit_definitions=>c_action-go_patch }{ lc_dummy_key }|
-      iv_class = |{ lc_action_class } { lc_online_class } | ).
+    lo_toolbar->add( iv_txt      = |Check|
+                     iv_act      = |{ zif_abapgit_definitions=>c_action-repo_code_inspector }{ lc_dummy_key }|
+                     iv_class    = |{ lc_action_class }|
+                     iv_li_class = |{ lc_action_class }| ).
 
-    ii_html->add( action_link( lv_patch_link && lc_separator ) ).
 
-    lv_diff_link = ii_html->a(
-      iv_txt   = |Diff|
-      iv_act   = |{ zif_abapgit_definitions=>c_action-go_repo_diff }{ lc_dummy_key }|
-      iv_class = |{ lc_action_class } { lc_online_class }| ).
+    lo_toolbar->add( iv_txt      = |Import|
+                     iv_act      = |{ zif_abapgit_definitions=>c_action-zip_import }{ lc_dummy_key }|
+                     iv_class    = |{ lc_action_class } { lc_offline_class }|
+                     iv_li_class = |{ lc_action_class }| ).
 
-    ii_html->add( action_link( lv_diff_link && lc_separator ) ).
+    lo_toolbar->add( iv_txt      = |Export|
+                     iv_act      = |{ zif_abapgit_definitions=>c_action-zip_export }{ lc_dummy_key }|
+                     iv_class    = |{ lc_action_class } { lc_offline_class }|
+                     iv_li_class = |{ lc_action_class }| ).
 
-    lv_check_link = ii_html->a(
-      iv_txt   = |Check|
-      iv_act   = |{ zif_abapgit_definitions=>c_action-repo_code_inspector }{ lc_dummy_key }|
-      iv_class = |{ lc_action_class }| ).
+    lo_toolbar->add( iv_txt      = |Settings|
+                     iv_act      = |{ zif_abapgit_definitions=>c_action-repo_settings }{ lc_dummy_key }|
+                     iv_class    = |{ lc_action_class }|
+                     iv_li_class = |{ lc_action_class }| ).
 
-    ii_html->add( action_link( lv_check_link && lc_separator ) ).
+    CREATE OBJECT lo_toolbar_more_sub EXPORTING iv_id = 'toolbar-ovp-more_sub'.
 
-    lv_zip_import_link = ii_html->a(
-      iv_txt   = |Import|
-      iv_act   = |{ zif_abapgit_definitions=>c_action-zip_import }{ lc_dummy_key }|
-      iv_class = |{ lc_action_class } { lc_offline_class }| ).
+    lo_toolbar_more_sub->add( iv_txt      = |Stage, filtered by Transport/Task|
```

Comment:
Just make the description shorter: Rename to "Export by Transport" and "Stage by Transport".

Reply:
@mbtools I renamed it.
![image](https://user-images.githubusercontent.com/59128840/150851291-d5007857-daf7-4d1d-a265-0bb3ef2c2dd2.png)

![image](https://user-images.githubusercontent.com/59128840/150851330-6f702e5a-38b8-42b1-a744-53848c015ffc.png)